js/main_nav.js

```diff
@@ -0,0 +1,26 @@
+let overlay = document.getElementById("main-nav");
+
+let navBurguer = document.getElementById("main-nav__burguer");
+
+let overlayMenu = document.getElementById("main-nav__menu-items");
+
+navBurguer.addEventListener("click", function () {
+	this.classList.toggle("burguer-animation");
+});
+
+navBurguer.addEventListener("click", function () {
+	overlay.classList.toggle("main-nav-overlay__animation");
+});
+
+navBurguer.addEventListener("click", function () {
+	overlayMenu.classList.toggle("main-nav__menu-items__animation");
```

Comment:
Could you use the previous eventListener to add this functionality?

Reply:
We talked about this offline before you sent it, please reach out to me if you are unsure on how to reduce this code.

Reply:
I just did it. Thank you both.